slaves/linux-cross/arm-linux-gnueabihf.config

```diff
@@ -0,0 +1,577 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Crosstool-NG Configuration
+#
+# Generated with crosstool-ng-1.22.0's `ct-ng menuconfig` command using the default configuration
+# plus the following modifications
+#
+# Target options > Target Architecture = arm
+# Target options > Architecture level = armv7-a -- (*)
```

Comment:
Hm that is a very good point. Right now we have no way of switching out the compiler for various cross compilations with the makefiles (that I know of), but I've been meaning to look harder into implementing `make dist` in rustbuild. Once we do that we should be able to switch over the cross builder to rustbuild I believe.

So I guess here's the ideal situation:

1. If easy, let's add an `armv7-linux-gnueabihf-gcc` to this docker image (configured appropriately)
2. We'll deploy this to prod, the makefiles just won't use `armv7-linux-gnueabihf-gcc`
3. Eventually when we switch to rustbuild we can configure to use `armv7-linux-gnuebihf-gcc`.

Although now that I think about it I'm not sure it matters all that much. This largely only affects glibc, and we're not distributing glibc, so...